packages/teams-js/test/private/externalAppAuthentication.spec.ts

```diff
@@ -593,6 +593,151 @@ describe('externalAppAuthentication', () => {
       }
     });
   });
+
+  describe('authenticateWithOauth2', () => {
+    const testOauthWindowParameters = {
+      width: 100,
+      height: 100,
+      isExternal: true,
```

Comment:
Should this be set to false for most unit tests?

Reply:
I think, the true/false value of isExternal does not affect the way teams-js send the value to host sdk. I have unit test with isExternal undefined.